Choreographer_30FPS/app/src/main/jni/Common/ndk_helper/JNIHelper.cpp

```diff
@@ -0,0 +1,750 @@
+/*
+ * Copyright 2013 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
maybe remove the Common folder:
  1) use static lib for native-activity
  2) use static lib for cpufeature [or pointing to the source in ndk]
  3) tuck ndk_helper to jni/ndk_helper directory 
might be easier for people to under.  

I feel teapot/moreteapot/Choreographer  should be within one android studio project, in which it builds for 3 apps. So inside ndk samples, only one teapot themem